components/fxa-client/src/config.rs

```diff
@@ -219,6 +227,7 @@ mod tests {
             issuer: "https://dev.lcip.org/".to_string(),
             jwks_uri: "https://oauth-stable.dev.lcip.org/v1/jwks".to_string(),
             token_endpoint: "https://stable.dev.lcip.org/auth/v1/oauth/token".to_string(),
+            introspection_endpoint: "https://stable.dev.lcip.org/auth/v1/introspect".to_string(),
```

Comment:
I believe this needs to be `https://oauth-stable.dev.lcip.org/v1/introspect` currently; you're right that it *should* be exposed under `/auth/v1/introspect` but I don't think it currently is :-(

Reply:
Does this fetch openid-configuration so these values don't need to be hard coded? Like you point out elsewhere, we can add the introspect endpoint into openid-configuration pretty easily.

Reply:
We do have logic for fetching from that endpoint, yes.